### PR TITLE
Show filename, lineNo, ColumnNo and tagName on hover in repl preview

### PR DESCRIPTION
**Pull request by imsamtar**
Show filename, lineNo, ColumnNo and tagName on hover in repl preview...

Screenshot from 2021-04-13 16-39-42

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -34,7 +55,7 @@ export default [...['compiler', 'bundler'].map(x => ({
 		svelte({
 			compilerOptions: {
 				// enable run-time checks when not in production
-				dev: dev
+				dev: true
```
Comment (jozef-slezak) [inline rollup.config.js]: what ist the purposte of changing to `dev: true` in all cases

File: src/env.js [hunk 2]
```diff
@@ -1 +1,4 @@
+// required by tippy.js
```
Comment (jozef-slezak) [inline src/env.js]: You haven't used tippy.js after all, have you?

**Comment by imsamtar** [comment]
#### Updated files

- `src/env.js`
- `rollup.config.js`

File: src/Tippy.svelte [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+                let _top = 0;
+                while (element && element.tagName !== "BODY") {
+                    _left += element.offsetLeft;
+                    _top += element.offsetTop;
+                    element = element.parentElement;
+                }
+                console.log(loc);
+                left = _left + event.clientX;
+                top = _top + event.clientY;
+                text = `${loc.file} Line: ${loc.line + 1} Column: ${loc.column + 1} Tag: ${target.tagName.toLowerCase()}`;
```
Comment (jozef-slezak) [inline src/Tippy.svelte]: Please parametrize this hardcoded text. Let's have something like `<Tippy text=`. Visual only component and separated function that calculates the text.

Comment (imsamtar) [reply src/Tippy.svelte]: OK will do that

**Comment by imsamtar** [comment]
- Removed calculate text and position logic from Tippy.svelte

**Comment by imsamtar** [comment]
Please merge this pull request so I can use svelte-repl.

**Pull request merged**